corehq/apps/reports/standard/monitoring.py

```diff
@@ -5,66 +5,65 @@
 import datetime
 import math
 from collections import defaultdict, namedtuple
+import six
+from six.moves import map, range
+from six.moves.urllib.parse import urlencode
```

Comment:
+1 for isort

what "manual rearranging" did you do? Is it that six now lives with the standard lib imports? Seems like manually changing isorts output kind of defeats the purpose of using an automatic tool for cleanup?

Reply:
Yeah, I put six up there and I put the `reports` module at the bottom.  I'm not sure where six belongs, actually.  Normally I'd make all in-module imports relative, so they'd sort to the bottom automatically, but that doesn't seem to be the way most other devs operate, so I abstained.

> Seems like manually changing isorts output kind of defeats the purpose of using an automatic tool for cleanup?

In an ideal world, maybe, but only a few people use isort, and I don't think we have a common config for it - do all versions use the same rules?  I'd be okay with that rule if we had an agreed upon config that included things like dealing with our horrible module structure or other exceptions.

Reply:
fair points. maybe its worth adding some linter configs to the repo/try to get consensus on that sort of thing?

Reply:
Yeah, I'd be in favor of that.  FWIW, I asked the book club, and they were in favor of putting `six` with the other 3rd party imports, and at least Farid was in favor of leaving the `isort` output intact, save for perhaps moving `__future__` imports to their own lines.